nextflow.config

```diff
@@ -57,67 +125,71 @@ try {
 }
 
 profiles {
-  conda { process.conda = "$projectDir/environment.yml" }
+  conda {
+    process.conda = "$baseDir/environment.yml"
+    conda.createTimeout = '1 h'
```

Comment:
Nope, but a nice thing to have for all pipelines! So would be great if you could open up a small 1-line PR to the template on nf-core/tools so that it can be synchronised out for everyone 👍🏻